yb-voyager/cmd/constants.go

```diff
@@ -117,6 +117,29 @@ const (
 
 	HTML_EXTENSION = ".html"
 	JSON_EXTENSION = ".json"
+
+	//adding constants for docs link
+	ADDING_PK_TO_PARTITIONED_TABLE_DOC_LINK = "https://deploy-preview-23634--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/known-issues/postgresql/#adding-primary-key-to-a-partitioned-table-results-in-an-error"
```

Comment:
nit: it seems the prefix will be similar(after deployment also) - 
```
https://deploy-preview-23634--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/known-issues/postgresql/
```

you can use defined a common, smaller variable for that part. 
That will be readable and can be easily modified when required.